packages/components-react/src/Hero.tsx

```diff
@@ -0,0 +1,60 @@
+import clsx from 'clsx';
+import { ForwardedRef, forwardRef, HTMLAttributes, PropsWithChildren, ReactNode } from 'react';
+import { Heading } from './Heading';
+import { Image } from './Image';
+import { Paragraph } from './Paragraph';
+
+export interface HeroProps extends HTMLAttributes<HTMLDivElement> {
+  alignment?: 'inline-start' | 'inline-end';
+  imageSrc: string;
+  imageAlt: string;
+  heading: ReactNode;
+  headingLevel?: 1 | 2 | 3 | 4 | 5 | 6;
+  subtext: ReactNode;
+  aspectRatio?: '16:9' | '1:1' | '4:3';
```

Comment:
Je kunt overwegen om de aspectRatio value gelijk te trekken met CSS
```suggestion
  aspectRatio?: '16 / 9' | '1 / 1' | '4 / 3';
```